openhim-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3'
```

Comment:
Did you run `mvn clean install`? I see this file does not have a header

Reply:
ya i have done it